packages/ripple-tide-grant/index.vue

```diff
@@ -0,0 +1,148 @@
+<template>
```

Comment:
I used `ripple-tide-landing-page` as a template for this, not sure how much of this markup is relevant?